### PR TITLE
chore: Updated analysers to v8 and fixed all errors (from new analyzers)

### PR DESCRIPTION
**Pull request by nickrandolph**
What kind of change does this PR introduce?
- Build or CI related changes

## What is the current behavior?

Warning that there's a new version of the net analyzers

## What is the new behavior?

Updated to latest analyzers(v8.0.0) and fixed build errors

## PR Checklist

Please check if your PR fulfills the following requirements:
- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Tested the changes where applicable:
	- [ ] UWP
	- [ ] WinUI
	- [ ] iOS
	- [ ] Android
	- [ ] WASM
	- [ ] MacOS
- [ ] Updated the documentation as needed:
	- [ ] [General Doc Update](https://github.com/unoplatform/uno.toolkit.ui/tree/main/doc)
	- [ ] [Controls Doc Update](https://github.com/unoplatform/uno.toolkit.ui/tree/main/doc/controls)
	- [ ] [Extensions Doc Update](https://github.com/unoplatform/uno.toolkit.ui/tree/main/doc/helpers)
	- [ ] [controls-styles.md](https://github.com/unoplatform/uno.toolkit.ui/blob/main/doc/controls-styles.md)
	- [ ] [lightweight-styling.md (LightWeight Styling Resource Keys)](https://github.com/unoplatform/uno.toolkit.ui/blob/main/doc/lightweight-styling.md)
- [ ] [Runtime Tests and/or UI Tests](https://platform.uno/docs/articles/contributing/guidelines/creating-tests.html) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal)
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/library/Uno.Toolkit.Cupertino/Styles/Controls/SegmentedControl.xaml [hunk 1]
```diff
@@ -44,8 +44,9 @@
     <StaticResource x:Key="CupertinoSegmentedControlItemForegroundSelectedPressed"
 							ResourceKey="CupertinoSystemBackgroundBrush" />
 
-    <StaticResource x:Key="CupertinoSegmentedControlItemBorderBrush"
-							ResourceKey="CupertinoBlueBrush" />
+	<!-- Which of these CupertinoSegmentedControlItemBorderBrush is correct -->
+    <!--<StaticResource x:Key="CupertinoSegmentedControlItemBorderBrush"
+							ResourceKey="CupertinoBlueBrush" />-->
```
Comment (kazo0) [inline src/library/Uno.Toolkit.Cupertino/Styles/Controls/SegmentedControl.xaml:49]: Can remove this

File: src/library/Uno.Toolkit.Cupertino/Styles/Controls/BottomTabBar.xaml [hunk 2]
```diff
@@ -37,11 +37,12 @@
...
     <Thickness x:Key="CupertinoBottomTabBarItemContentOnlyMargin">12,0</Thickness>
 
+	<!-- Which of these CupertinoBottomTabBarBackground resources is correct -->
     <StaticResource x:Key="CupertinoBottomTabBarBackground"
 							ResourceKey="CupertinoBottomTabBarBackgroundBrush" />
 
-    <StaticResource x:Key="CupertinoBottomTabBarBackground"
-							ResourceKey="SystemControlTransparentBrush" />
+    <!--<StaticResource x:Key="CupertinoBottomTabBarBackground"
+							ResourceKey="SystemControlTransparentBrush" />-->
```
Comment (kazo0) [inline src/library/Uno.Toolkit.Cupertino/Styles/Controls/BottomTabBar.xaml:45]: Can remove

**Review by kazo0 — APPROVED** [review]

File: src/library/Uno.Toolkit.Cupertino/Styles/Controls/BottomTabBar.xaml (hunk 2 above)
Comment (nickrandolph) [reply src/library/Uno.Toolkit.Cupertino/Styles/Controls/BottomTabBar.xaml]: done

File: src/library/Uno.Toolkit.Cupertino/Styles/Controls/SegmentedControl.xaml (hunk 1 above)
Comment (nickrandolph) [reply src/library/Uno.Toolkit.Cupertino/Styles/Controls/SegmentedControl.xaml]: done

**Pull request merged**